try{
    console.log(rahul)
}catch(error){
    console.log(error.name)
}